triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -60,8 +59,8 @@ trait AbstractTriggerServiceTest
   protected val darPath = requiredResource("triggers/service/test-model.dar")
 
   // Encoded dar used in service initialization
-  protected val dar = DarReader.assertReadArchiveFromFile(darPath).map(p => p.pkgId -> p.proto)
-  protected val testPkgId = dar.main._1
+  protected lazy val dar = DarReader.assertReadArchiveFromFile(darPath).map(p => p.pkgId -> p.proto)
+  protected lazy val testPkgId = dar.main._1
```

Comment:
These lazy changes allow the `darPath` to be modified by subclassing - e.g. so that version specific projects may be loaded by testing